foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/util/MavenUtil.java

```diff
@@ -221,7 +222,7 @@ public String getPackaging() {
      * @param context GenerationContext
      * @return the profile for pyspark data delivery pipeline
      */
-    public static String getPySparkDataDeliveryProfileName(GenerationContext context) {
+    public static boolean isRecordGenerationInPipelineModule(GenerationContext context) {
```

Comment:
a: The java docs should be updated to reflect the change in return value and function expectation